api/src/pcapi/templates/admin/booking.html

```diff
@@ -28,11 +28,11 @@ <h2>Réservations</h2>
 <p>
     Si cette réservation a été annulée par erreur (ou
     frauduleusement) alors qu'elle a en fait été utilisée, il est
-    possible de la marquer comme utilisée.
+    possible de la Désannuler.
```

Comment:
pas de majuscule au milieu d'une phrase

```suggestion
    possible de la désannuler.
```

idem dans les autres fichiers